src/testUtils/java/org/broadinstitute/hellbender/testutils/IntegrationTestSpec.java

```diff
@@ -63,6 +64,7 @@ public IntegrationTestSpec(String args, List<String> expectedFileNames) {
         this.compareBamFilesSorted = false;
         this.validationStringency = ValidationStringency.DEFAULT_STRINGENCY;
         this.tempExtension = DEFAULT_TEMP_EXTENSION;
+        this.trimWhiteSpace = true;
```

Comment:
Was `IntegrationTestSpec` previously defaulting to trimming whitespace before doing the comparison vs the expected output(s), or is this PR introducing a change in default behavior?

Reply:
@droazen previously (and still after this pr) IntegrationTestSpec defaulted to trimming whitespace. Which is how this bug was never caught by tests before.

Reply:
Ah, I see...in that case I think I'll open a separate ticket to see if we can change the default for `IntegrationTestSpec` to not ignore leading/trailing whitespace. Go ahead and merge once tests pass 👍